docs/content/how-to/aws/index.md

```diff
@@ -0,0 +1,555 @@
+---
+title: AWS Architecture and Resources
+---
+
+# AWS Architecture and Resources
+
+In this section we will talk about the AWS Resources and Openshift objects generated by Hypershift and how the Controllers interact with them during a Hosted Cluster deployment.
+
+## Prerequisites and Assumptions
+
+Here we will set the legend about the terms used in this document:
```

Comment:
Please don't add another source of truth, let's refer to https://deploy-preview-1863--hypershift-docs.netlify.app/reference/concepts-and-personas/ and add what's missing there.